cve_bin_tool/cli.py

```diff
@@ -197,28 +194,24 @@ def scan_file(self, filename):
                         version = result["version"]
                     elif result["version"] == "UNKNOWN":
                         self.logger.warning(
-                            "%r was detected with version UNKNOWN in file %r"
-                            % (result["modulename"], filename)
+                            f"{modulename} was detected with version UNKNOWN in file {filename}"
                         )
                     else:
-                        self.logger.error(
-                            "No version info for %r", result["modulename"]
-                        )
+                        self.logger.error(f"No version info for {modulename}")
 
                     if version != "UNKNOWN":
                         found_cves = self.get_cves(vendor_package_pairs, version)
                         if found_cves:
                             self.files_with_cve = self.files_with_cve + 1
                         self.all_cves[modulename][version] = found_cves
                         self.logger.info(
-                            "%s %s %s %s"
-                            % (filename, result["is_or_contains"], modulename, version)
+                            f'{filename} {result["is_or_contains"]} {modulename} {version}'
                         )
                         if found_cves.keys():
-                            self.logger.info("Known CVEs in version " + str(version))
```

Comment:
Wow, some of our string format choices were weird.  I'm guessing this str() was in there for 2.7 compatibility or leftover from one of the earlier versions of the tool.